connector/src/main/java/org/jboss/as/connector/subsystems/jca/JcaExtension.java

```diff
@@ -630,7 +641,14 @@ private void parseDistributedWorkManager(final XMLExtendedStreamReader reader, f
                     }
                     case ELYTRON_ENABLED: {
                         switch (readerNS) {
-                            case JCA_5_0: {
+                            case JCA_5_0:
+                            {
+                                String value = rawElementText(reader);
+                                ((SimpleAttributeDefinition) JcaDistributedWorkManagerDefinition.DWmParameters.ELYTRON_ENABLED_5_0.getAttribute()).parseAndSetParameter(value, distributedWorkManagerOperation, reader);
+                                break;
```

Comment:
Same comment I made in parseWorkManager applies here as well.

Reply:
fixed